35/javadoc/allclasses-index.html

```diff
@@ -655,7 +656,7 @@ <h1 title="All Classes and Interfaces" class="title">All Classes and Interfaces<
 <div class="col-first odd-row-color all-classes-table all-classes-table-tab5"><a href="org/apache/kafka/common/errors/CoordinatorLoadInProgressException.html" title="class in org.apache.kafka.common.errors">CoordinatorLoadInProgressException</a></div>
 <div class="col-last odd-row-color all-classes-table all-classes-table-tab5">
 <div class="block">In the context of the group coordinator, the broker returns this error code for any coordinator request if
- it is still loading the group metadata (e.g.</div>
+ it is still loading the group metadata (e.g. after a leader change for that group metadata topic partition).</div>
```

Comment:
Good catch, do we know why it got cut previously?

Reply:
Seriously, I have no idea.